src/api/project/controllers/project.js

```diff
@@ -4,82 +4,29 @@ const { createCoreController } = require('@strapi/strapi').factories;
 
 module.exports = createCoreController('api::project.project', ({ strapi }) => ({
     async findOne(ctx) {
-        const { slug } = ctx.params;
+        const slug = ctx.request.params.id;
```

Comment:
This was the fix. See https://docs.strapi.io/dev-docs/backend-customization/requests-responses#ctxrequest.